### DISCUSSION TITLE
Adding metadata to tasks that are already saved in taskwarrior.

### DISCUSSION THREAD

**Discussion by fullstopslash** [tbabej/taskwiki#379] [Community support]
So I'm familiar with the annotation to add metadata for a _new_ task that's being created. ie.
* [ ] Task -- +@context +Tag

My question is this: Is there currently a way to add metadata to a multitude of tasks that are **already saved** in taskwarrior. Because currently it seems that if I edit the contents of the task TaskWiki will simply add the tag to the title of the task and not add the tag to the **tags** section. Example being if I do something like this to Task.

* [ ] Task #abcdef
(edit)
* [ ] Task -- +@context +Tag #abcdef

**Comment by liskin** [comment]
Yes, there's `:TaskWikiMod` and it's bound to `,tm`. You can select several tasks and then type `,tm` and they'll all be modified.
